students/12th/najieun/westagram/js/login.js

```diff
@@ -0,0 +1,27 @@
+const button = document.getElementsByClassName('loginBtn')[0];
+const name = document.getElementsByClassName("inputName")[0];
+const pw = document.getElementsByClassName("inputPw")[0];
+
+function changeColor() {
+    const nameValue = name.value;
+    const pwValue = pw.value;
+
+    if (nameValue && pwValue.length >= 6) {
+        button.removeAttribute('disabled');
+        button.style.backgroundColor = '#0095F6';
+    }
+    else {
+        button.setAttribute("disabled", "")
+        button.style.backgroundColor = '#B9E0FC';
+    }
```

Comment:
```suggestion
    const isVaild = nameValue && pwValue.length >= 6
    if (isValid) {
        button.removeAttribute('disabled');
        button.style.backgroundColor = '#0095F6';
    } else {
        button.setAttribute("disabled", "")
        button.style.backgroundColor = '#B9E0FC';
    }
```
- else문 줄바꿈해서 사용하면 안됩니다.
- 조건에 이름을 붙여주면 훨씬 읽기 좋아집니다

Reply:
else 줄바꿈 X
조건에 이름 붙여줬습니다~~